scrunch/expressions.py

```diff
@@ -475,20 +475,53 @@ def category_ids(var_id, var_value, variables=variables):
                 return var_value
             return value
 
+        def adapt_multiple_response(var_url, values):
+            # if variable is multiple_response
+            #       convert value --> column and change ids by aliases
+            var_id = variable_id(var_url)
+            aliases = []
+            for var in variables:
+                if variables[var]['id'] == var_id:
+                    for sub_var_id in variables[var]['subreferences']:
```

Comment:
This info is available on the variable resource. On it you have `.subvariables` a list of ordered subvariables and `.subreferences` a dictionary keyed by subvariable url and the references for each so you can access `array_var_res.body.subreferences[suvariable_url]['alias']` in straight access